open-metadata-implementation/integration-services/organization-integrator/organization-integrator-api/build.gradle

```diff
@@ -9,6 +9,8 @@ dependencies {
     implementation project(':open-metadata-implementation:frameworks:open-connector-framework')
     implementation project(':open-metadata-implementation:governance-servers:integration-daemon-services:integration-daemon-services-api')
     implementation project(':open-metadata-implementation:repository-services:repository-services-apis')
+    implementation project(':open-metadata-implementation:access-services:community-profile:community-profile-api')
```

Comment:
in general our we have both server & client dependent on the api module, and api not dependent on either - I notice here the api is dependent on client. was this intentional?